packages/faustwp-core/README.md

```diff
@@ -0,0 +1,3 @@
+# Faustwp core
```

Comment:
```suggestion
# @faustwp/core
```

Reply:
Note that the [current core package titles it's README as @faustjs/core](https://www.npmjs.com/package/@faustjs/core).

We may want to add another README.md to faustwp-cli for consistency.